File: 20221109-LifeBoat-java/app/src/test/java/LifeBoatTest.java
import org.junit.jupiter.api.*;

import static org.junit.jupiter.api.Assertions.*;

class LifeBoatTest {
  @Test
  void simple() {
    LifeBoat lifeBoat = new LifeBoat();

  //  assertEquals(3,lifeBoat.solution(new int[] {70,50,80,50},100));
    assertEquals(3,lifeBoat.solution(new int[] {70,80,50},100));
  }

}

File: 20221110-LifeBoat-javaScript/LifeBoat.test.js
// 내 것과 당첨 표  일치번호 갯수구하기
// 0번호 갯수에 따른 최대 최소 구기

import { sort } from './LifeBoat';

test('sort', () => {
  expect(sort([70, 50, 80, 50]))
    .toStrictEqual([50, 50, 70, 80]);
});
